### DISCUSSION TITLE
SwiftUI MKWebview not handling gestures

### DISCUSSION THREAD

**Discussion by CarlSchwarz** [chartjs/chartjs-plugin-zoom#869] [Q&A]
I'm not sure if this is an issue with the zoom plugin or with my MKWebView setup in SwiftUI but I made a chart that works everywhere except in MKWebView. If instructions for MKWebView could also be included somewhere that would be good, otherwise if it doesn't work I'm alerting a bug.
https://stackoverflow.com/questions/74344519/the-chart-js-zoom-and-pan-plugin-is-not-working-in-wkwebview

**Comment by kurkle** [comment]
I'm guessing the MKWebView captures the gestures and those are not passed to hammer/zoom.
